src/tagsouppullparser.cpp

```diff
@@ -75,7 +75,7 @@ TagSoupPullParser::Event TagSoupPullParser::next()
 	case Event::START_DOCUMENT:
 	case Event::START_TAG:
 	case Event::END_TAG:
-		skip_whitespace();
+		inputstream->read(&c, 1);
```

Comment:
I think this change makes bda17e468e17e3e993c854a6a1af282673136a2b unnecessary, and it also means that 72a70b4774317f1928f6d82ebbd7383b63c0dc64 could be squashed into this commit.

Reply:
Done.
I've force-pushed after an interactive rebase.

To make sure the diff can still be seen, I've temporarily added a branch which points to the commit before the force push.
This shows that the end result is identical (just the commits have been reordered/combined): https://github.com/newsboat/newsboat/compare/234e26ac3a317e64bea23766ea0462f6a645772d..404c0896327224e5203f77c339984fe8be9b32e0